modules/coreboot: set Dasharo coreboot fork rev to the main dasharo branch

mkopec opened the pull request:
Here are the changes (mostly irrelevant to Heads) that were merged after https://github.com/linuxboot/heads/pull/1846:
- Support for capsule updates (inactive for Heads build)
- Added NovaCustom MTL dGPU variants
- EC firmware update in coreboot (inactive for Heads build)

Changes that affect V560TU:
- Added support for reading power adapter wattage via ACPI
- Fixes for USB not working
- Add AC/DC loadline programming, for better stability
- Fixes for ME disable
- Fixes for Ethernet detection

I've tested booting into Qubes, ethernet detection, and USB type-A.


tlaurion edit: took https://github.com/linuxboot/heads/pull/1889#issuecomment-2598297536 content

mkopec commented:
@tlaurion Ideally https://github.com/Dasharo/coreboot/pull/609 shouldve been merged before https://github.com/linuxboot/heads/pull/1846 , but it went the other way around, so I'm bumping the coreboot to what was merged in the end: https://github.com/Dasharo/coreboot/commits/048ca832325d716fcab596822b10f5d493fc2312/

macpijan commented:
Can we explain briefly here in heads upstream MR what has changed since that revision, and which models are affected?

mkopec commented:
Mostly changes irrelevant to Heads that were merged in the meantime:
- Support for capsule updates (inactive for Heads build)
- Added NovaCustom MTL dGPU variants
- EC firmware update in coreboot (inactive for Heads build)

Changes that affect V560TU:
- Added support for reading power adapter wattage via ACPI
- Fixes for USB not working
- Add AC/DC loadline programming, for better stability
- Fixes for ME disable
- Fixes for Ethernet detection

tlaurion commented:
This is the changeset between pointed commit change 
https://github.com/Dasharo/coreboot/compare/94e5f5d5b808cf8d8fd5c70d4ef6a08a054f8986...048ca832325d716fcab596822b10f5d493fc2312

tlaurion commented:
@mkopec 

- [ ]  just received v560tu.
- [ ] tested on nv41 confirmed pr0 works
- [ ] tested on v560tu confirming previous changes and pr0 works
- [ ] merge

tlaurion commented:
Something is wrong with second stage QubesOS install, i'm stuck at installing debian-12-xfce template for more than an hour and none of the available terminals provide any interesting output.

Since I can boot into the second stage installer and none of this is related to proposed changes, I'll merge.

tlaurion reviewed (approved):
Merging

tlaurion commented:
> Something is wrong with second stage QubesOS install, i'm stuck at installing debian-12-xfce template for more than an hour and none of the available terminals provide any interesting output.
> 
> Since I can boot into the second stage installer and none of this is related to proposed changes, I'll merge.

Unfortunately, my nv41 did an "instant reboot" today. I have to revert the changes on master and #1875 @mkopec
